src/test/resources/com/google/jenkins/plugins/computeengine/configuration-as-code.yml

```diff
@@ -15,9 +15,6 @@ jenkins:
             numExecutorsStr:    1
             runAsUser:          jenkins
             remoteFs:           agent
-            windows:            false
```

Comment:
Should we add a separate test case in the CASC tests for windows configs?

Reply:
Yes, definitely